definitions/features/foreman_tasks.rb

```diff
@@ -93,10 +93,8 @@ def delete(state)
       DELETE FROM foreman_tasks_locks as ftl where ftl.task_id NOT IN (SELECT id FROM foreman_tasks_tasks);
     SQL
 
-    if check_min_version(foreman_plugin_name('foreman-tasks'), '4.0.0')
-      sql += 'DELETE FROM foreman_tasks_links as ftl ' \
-             'where ftl.task_id NOT IN (SELECT id FROM foreman_tasks_tasks);'
-    end
+    sql += 'DELETE FROM foreman_tasks_links as ftl ' \
+           'where ftl.task_id NOT IN (SELECT id FROM foreman_tasks_tasks);'
```

Comment:
should we just roll this into the `sql` defined in L87-94? no need to `+=` IMHO.